configs/mainnet.json

```diff
@@ -80,5 +80,54 @@
             "denom": "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518",
             "amount": "600527806000000"
         }
-    ]
-}
+    ],
+    "transfer_options": {
+      "Cosmos Hub": 0,
+      "Osmosis": 1,
+      "Evmos": 2,
+      "Axelar": 3,
+      "Juno": 4,
+      "Stride": 5,
+      "Kujira": 6,
+      "Terra": 7,
+      "Comdex": 8,
+      "Stafihub": 9,
+      "Persistence Core": 10,
+      "Stargaze": 11,
+      "Canto": 12,
+      "Sommelier": 13,
+      "Ethereum": 14,
+      "Arbitrum": 15,
+      "Polygon": 16,
+      "OKC": 17,
+      "Binance Smart Chain": 18,
+      "Carbon": 19,
+      "Neo": 20,
+      "Neo3": 21,
+      "Zilliqa": 22
+    },
+    "network_fees": {
+      "swth": 0,
+      "usc": 1,
+      "ibc/A4DB47A9D3CF9A068D454513891B526702455D3EF08FB9EB558C561F9DC2B701": 2,
+      "ibc/ED07A3391A112B175915CD8FAF43A2DA8E4790EDE12566649D0C2F97716B8518": 3,
+      "ibc/2B58B8C147E8718EECCB3713271DF46DEE8A3A00A27242628604E31C2F370EF5": 4,
+      "ibc/662914D0C1CEBCB070C68F061D035E8B10A07C79AB286E7342C85F3BE74612C5": 5,
+      "ibc/3552CECB7BCE1891DB6070D37EC6E954C972B1400141308FCD85FD148BD06DE5": 6,
+      "ibc/92E974290AF9E2BC3AEEEC35305C8FD76AC5A22A74CF8D91270FDF5A1C41E861": 7,
+      "ibc/35E771B8682D828173F4B795F6C307780F96DC64D6F914FAE4CC9B4666F66364": 8,
+      "ibc/07FA7831E1920D0C87C9388F86B0108677F6ED0C9DE7E4063F05ED675192405C": 9,
+      "ibc/75249A18DEFBEFE55F83B1C70CAD234DF164F174C6BC51682EE92C2C81C18C93": 10,
+      "ibc/B7864B03E1B9FD4F049243E92ABD691586F682137037A9F3FCA5222815620B3C": 11,
+      "bnb.1.6.773edb": 12,
+      "usdc.1.6.53ff75": 13,
+      "busd.1.6.754a80": 14,
+      "cgt/1": 15,
```

Comment:
Suggestion: Think it would be good to bump up `cgt/1` to above USC as it is a Carbon token and possibly the token with the most quantity (auto-conversion in 1:1 ratio).